PROCESS.md

```diff
@@ -0,0 +1,119 @@
+   brought to the TSC prior to and during development.
+3. Feature proposal issues are added to the 
+   [Roadmap](https://github.com/AcademySoftwareFoundation/OpenColorIO/projects/3) 
+   project in GitHub by a committer for tracking the status of all accepted 
+   proposals in one place.
+4. During development, work in progress may optionally be shared for feedback 
+   and early review via a 
+   [draft pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests).
+5. Upon completion of feature development, a pull request is created for review 
+   and 
+   [linked to the associated issue]((https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) for review.
+   Linking the issue and PR will result in the issue being closed automatically 
+   upon the PR being merged, and facilitate tracking of the proposed feature in
+   the 
+   [Roadmap](https://github.com/AcademySoftwareFoundation/OpenColorIO/projects/3). 
+   See [CONTRIBUTING.md](CONTRIBUTING.md) for an overview of commit signing 
+   requirements and a guide on submitting a pull request.
+6. The pull request is merged only after obtaining 2 approvals from committers 
+   listed in [COMMITTERS.md](COMMITTERS.md) that are not affiliated with the 
+   author or their company. If approval does not happen within two weeks and 
+   there are no raised objections or unresolved discussions happening about the 
+   feature, a committer that is affiliated with the author may approve it. See 
+   [Required Approvals](#Required-Approvals) for a detailed breakdown of OCIO 
+   pull request approval policy.
+
+**NOTE**: Software design is often an iterative process with development. While 
+a design proposal is required, it is understood that development of the 
+proposed feature may begin prior to opening a design proposal issue in GitHub. It 
+is however highly encouraged to propose the design early in the interest of 
+respecting the contributor's time and effort. This avoids getting too far into 
+development without community involvement, which may lead to redesign and 
+extensive rework before changes can be approved.
+
+## Required Approvals
+
+Modifications of the contents of the OpenColorIO repository are made on a
+collaborative basis. Anyone with a GitHub account may propose a modification via
+pull request and it will be considered by the project Committers.
+
+Pull requests must meet a minimum number of Committer approvals prior to being
+merged. Rather than having a hard rule for all PRs, the requirement is based on
+the complexity and risk of the proposed changes, factoring in the length of
+time the PR has been open to discussion. The following guidelines outline the
+project's established approval rules for merging:
+
+* Core design decisions, large new features, or anything that might be perceived
```

Comment:
This part should be edited or removed as it overlaps with the previous section.  Let's keep this section focused on the Required Approvals and the previous one on the feature development process.

Reply:
Revised.